streamlit/streamlit_code_backup_1_02_(10_4_22).py

```diff
@@ -0,0 +1,639 @@
+    page_title="LENSS Sensor Plotting",
+    page_icon="🌠",
+    initial_sidebar_state="expanded",
+)
+
+
+# Login Page, only allows access once the correct password is entered
+with st.form("Login"):
+    input_password = st.text_input("Please Input Password", "")
+    submitted = st.form_submit_button("Submit")
+
+if input_password == "Y3rke5*":
+    st.success("Correct Password!")
+else:
+    st.stop()
+
+
+# Page setup that is only visible once the password is entered
+st.title("LENSS Sensor Plotting")
+st.subheader("Configure the plot")
+# Night/Sensor selection
+sensor_selection = st.selectbox(
+    label="Choose a Sensor", options=("Sensor 5", "Sensor 11", "Sensor 12")
+)
+st.caption("Listed sensors have all available data from 2022-06-01 to today")
+day_selection = st.date_input(
+    "Choose a Night", datetime.date(datetime.now() - timedelta(2))
+)
+# Finds out the current date
+morning_selection = day_selection + timedelta(days=1)
+
+
+# Prepares variable names, depending on selection, for later
+if sensor_selection == "Sensor 5":
+    filename = str(day_selection) + "_LENSSTSL0005"
+    filename2 = str(morning_selection) + "_LENSSTSL0005"
+    sensornumber = "05"
+    filetype = "Old"
+
+if sensor_selection == "Sensor 11":
+    filename = str(day_selection) + "_LENSSTSL0011"
+    filename2 = filename2 = str(morning_selection) + "_LENSSTSL0011"
+    sensornumber = "11"
+    filetype = "Old"
+
+if sensor_selection == "Sensor 12":
+    filename = str(day_selection) + "_LENSSTSL12"
+    filename2 = str(morning_selection) + "_LENSSTSL12"
+    sensornumber = "12"
+    filetype = "New"
+
+    # Imports files
+morning_error = False
+night_error = False
+if filetype == "New":
+    try:
+        sensor = pandas.read_csv(
+            "files/" + filename + ".txt",
+            sep=";",
+            names=[
+                "UTC",
+                "CST",
+                "Temperature",
+                "Frequency",
+                "Voltage",
+                "Sensor Number",
+            ],
+        )
+    except:
+        morning_error = True
+    try:
+        sensor2 = pandas.read_csv(
+            "files/" + filename2 + ".txt",
+            sep=";",
+            names=[
+                "UTC",
+                "CST",
+                "Temperature",
+                "Frequency",
+                "Voltage",
+                "Sensor Number",
+            ],
+        )
+    except:
+        night_error = True
+else:
+    try:
+        sensor = pandas.read_csv(
+            "files/" + filename + ".txt",
+            names=["Time", "Voltage", "Frequency", "Temperature", "Sensors"],
+        )
+    except:
+        morning_error = True
+    try:
+        sensor2 = pandas.read_csv(
+            "files/" + filename2 + ".txt",
+            names=["Time", "Voltage", "Frequency", "Temperature", "Sensors"],
+        )
+    except:
+        night_error = True
+
+    # Feedback/Error messages if file is unavailable
+if morning_error == True:
```

Comment:
These can be changed to just `if morning_error:`, the `if morning_error == True` is redundant.